tests/queries/0_stateless/arcadia_skip_list.txt

```diff
@@ -1,6 +1,7 @@
 # Add testcase here to skip it in Arcadia CI (Yandex synchronization check)
 # It is useful for tests with not supported features in Arcadia build
 00105_shard_collations
+00184_shard_distributed_group_by_no_merge
```

Comment:
Just to note (since I'm not sure about it), I've added here already existing test, since it was modified (and the arcadia build may not support new bits)

Reply:
Ok.